plugins/hadoopsecuritymanager-common/src/azkaban/security/commons/HadoopSecurityManager.java

```diff
@@ -33,49 +34,42 @@
   public static final String PROXY_USER = "proxy.user";
   public static final String USER_TO_PROXY = "user.to.proxy";
   public static final String OBTAIN_BINARY_TOKEN = "obtain.binary.token";
-  public static final String MAPREDUCE_JOB_CREDENTIALS_BINARY =
-      "mapreduce.job.credentials.binary";
+  public static final String MAPREDUCE_JOB_CREDENTIALS_BINARY = "mapreduce.job.credentials.binary";
 
-  public static final String OBTAIN_JOBTRACKER_TOKEN =
-      "obtain.jobtracker.token";
+  public static final String OBTAIN_JOBTRACKER_TOKEN = "obtain.jobtracker.token";
   public static final String OBTAIN_NAMENODE_TOKEN = "obtain.namenode.token";
   public static final String OBTAIN_HCAT_TOKEN = "obtain.hcat.token";
 
-  public boolean isHadoopSecurityEnabled()
-      throws HadoopSecurityManagerException {
+  public boolean isHadoopSecurityEnabled() throws HadoopSecurityManagerException {
     return false;
   }
 
   /**
    * Create a proxied user based on the explicit user name, taking other
    * parameters necessary from properties file.
    */
-  public abstract UserGroupInformation getProxiedUser(String toProxy)
-      throws HadoopSecurityManagerException;
+  public abstract UserGroupInformation getProxiedUser(String toProxy) throws HadoopSecurityManagerException;
 
   /**
    * Create a proxied user, taking all parameters, including which user to proxy
    * from provided Properties.
    */
-  public abstract UserGroupInformation getProxiedUser(Props prop)
-      throws HadoopSecurityManagerException;
+  public abstract UserGroupInformation getProxiedUser(Props prop) throws HadoopSecurityManagerException;
 
-  public abstract FileSystem getFSAsUser(String user)
-      throws HadoopSecurityManagerException;
+  public abstract FileSystem getFSAsUser(String user) throws HadoopSecurityManagerException;
 
   public static boolean shouldProxy(Properties prop) {
     String shouldProxy = prop.getProperty(ENABLE_PROXYING);
 
     return shouldProxy != null && shouldProxy.equals("true");
   }
 
-  public abstract void prefetchToken(File tokenFile, String userToProxy,
-      Logger logger) throws HadoopSecurityManagerException;
-
-  public abstract void cancelTokens(File tokenFile, String userToProxy,
-      Logger logger) throws HadoopSecurityManagerException;
+  public abstract void prefetchToken(File tokenFile, String userToProxy, Logger logger)
+      throws HadoopSecurityManagerException;
 
```

Comment:
I don't think that there is any diff in this file ? If yes, then we should just remove it from diff.